### PR TITLE
Run3-TB62 Modify HGCal TB related scenarios and scripts in view of changed initialization of SDs

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify HGCal TB related scenarios and scripts in view of changed initialization of SDs

#### PR validation:

Use the scripts along with #34589

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by srimanob** [comment]
+Upgrade

File: SimG4CMS/HGCalTestBeam/test/HGCalTBCERN170_cfg.py [hunk 1]
```diff
@@ -95,16 +95,10 @@
 process.VtxSmeared.MaxY =  7.5
 process.g4SimHits.HGCSD.RejectMouseBite = True
 process.g4SimHits.HGCSD.RotatedWafer    = True
-process.g4SimHits.OnlySDs = ['HGCalSensitiveDetector',
```
Comment (perrotta) [inline SimG4CMS/HGCalTestBeam/test/HGCalTBCERN170_cfg.py:98]: Out of curiosity, which is the difference between 'HGCalSensitiveDetector' and 'HGCSensitiveDetector'?

**Comment by perrotta** [comment]
+1

**Pull request merged**